source/server/core_server.cpp

```diff
@@ -1,9 +1,12 @@
+
 #include <niscope/niscope_library.h>
 #include <niscope/niscope_service.h>
 #include <niswitch/niswitch_library.h>
 #include <niswitch/niswitch_service.h>
 #include <nisync/nisync_library.h>
 #include <nisync/nisync_service.h>
+#include <nidigital/nidigital_library.h>
```

Comment:
Can you add these two lines towards beginning of file to keep the include statements sorted in alphabetical order.

Reply:
fixed this